compiler/rustc_typeck/src/check/upvar.rs

```diff
@@ -626,7 +633,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
             self.tcx.struct_span_lint_hir(
                 lint::builtin::RUST_2021_INCOMPATIBLE_CLOSURE_CAPTURES,
                 closure_hir_id,
-                closure_head_span,
+                 closure_head_span,
```

Comment:
I am a bit surprised that rustfmt isn't complaining about this extra space.

Reply:
I'm surprised `./x.py fmt` made this change